jobs/scheduled/create_holiday_events.rb

```diff
@@ -75,14 +75,18 @@ def execute(args)
                   holiday[:date]
                 end
 
-              CalendarEvent.find_or_create_by(
-                topic_id: topic_id,
-                user_id: user_id,
-                username: usernames[user_id],
-                description: holiday[:name],
-                start_date: date,
-                region: region,
-              )
+              event =
+                CalendarEvent.find_or_create_by(
+                  topic_id: topic_id,
+                  user_id: user_id,
+                  username: usernames[user_id],
+                  description: holiday[:name],
+                  start_date: date,
+                  region: region,
+                )
+
+              # TODO: Merge update! to find_or_create_by after 06-2024.
```

Comment:
What is the context of this todo? Would you be creating a topic + timer for it?

Reply:
I had brain fog :)